src/status_im/common/bottom_sheet/view.cljs

```diff
@@ -106,7 +106,14 @@
                     {:transform [{:translateY translate-y}]}
                     (style/sheet insets window-height selected-item))}
            (when shell?
-             [blur/ios-view {:style style/shell-bg}])
+             [blur/ios-view
+              (cond-> {:style style/shell-bg}
+                blur-bg? (assoc :blur-radius
```

Comment:
This `assoc` call is kind of unreadable with each argument on its own line instead of each key & value pair on each line.

Reply:
Maybe we can move the `assoc` next line.